language/stdlib/modules/LibraConfig.move

```diff
@@ -72,19 +72,41 @@ module LibraConfig {
         reconfigure_();
     }
 
+    // Set a config item to a new value and trigger a reconfiguration.
+    public fun set_with_capability_and_reconfigure<Config: copyable>(
+        _cap: &ModifyConfigCapability<Config>,
+        payload: Config
+    ) acquires LibraConfig, Configuration {
+        let addr = CoreAddresses::LIBRA_ROOT_ADDRESS();
+        assert(exists<LibraConfig<Config>>(addr), ECONFIG_DOES_NOT_EXIST);
+        let config = borrow_global_mut<LibraConfig<Config>>(addr);
+        config.payload = payload;
+        reconfigure_();
+    }
+
+    // Publish a new config item.
+    // The caller will use the returned ModifyConfigCapability to specify the access control
+    // policy for who can modify the config.
+    // Does not trigger a reconfiguration.
+    public fun publish_new_config_and_get_capability<Config: copyable>(
```

Comment:
sorry I probably miss something, looking at this again, the original code should work as is? what's the motivation of creating these two methods?

the config account holds the `ModifyConfigCapability` and is able to call `set::<LibraSystem>` directly, no?

Reply:
ah nvm I see, it enables the operator account to borrow the capability